pos_order_remove_line/static/src/js/orderline.js

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright 2023 LevelPrime
+ *  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+ */
+
+odoo.define("pos_order_remove_line.Orderline", function (require) {
```

Comment:
I think it was ok to move to ESM module style as Odoo is transitioning it as well. The only issue was to rename the file so pre-commit would swallow it...

Reply:
Sure, I will leave it as it was before.
Thanks.

Reply:
Changes done!